force-app/main/default/objects/npe01__OppPayment__c/fields/RefundType__c.field-meta.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>RefundType__c</fullName>
+    <description>Determine if the record is fully or partially refunded</description>
+    <externalId>false</externalId>
+    <inlineHelpText>Determine if the record is fully or partially refunded</inlineHelpText>
+    <label>Refund Type</label>
+    <required>false</required>
+    <trackFeedHistory>false</trackFeedHistory>
+    <trackTrending>false</trackTrending>
+    <type>Picklist</type>
+    <valueSet>
+        <valueSetDefinition>
+            <sorted>false</sorted>
+            <value>
+                <fullName>Refund</fullName>
```

Comment:
@force2b @andrewyu-salesforce We have a cross-team meeting on this field later today. It is likely we will be asked to use the Elevate Type (text) field, and add Partial Refund and Chargeback as field values. I will update here when we have consensus.

Reply:
@force2b, @andrewyu-salesforce, @lparrott We have decided to go with the picklist. @melissabarber will get the text finalized in UI text review tomorrow. I'll update ASAP.

Reply:
The field is repurposed to show if a payment record is a `Refund`, `Partial Refund`, or `Charge Back` record instead of showing if the Original Payment has been refunded. The decision is made to not update anything on the Original Payment record.